specs/1137-CLNT-client_api.md

```diff
@@ -0,0 +1,45 @@
+# Client API
+
+## Methods
+
+### Connect (client.connect_wallet)
+
+- If I connect and it is not approved then I get an error telling me the user rejected the connection
+- If I connect and it is approved then I do not get an error
+- If I connect and it is approved and I try to connect again then the connection is automatically approved
```

Comment:
Is there a time limit or anything on this?

With me being new and dumb to browser wallet, this semi reads that if a connection as been approved once, any future connections will never need to be re approved?

All good if that's actually the case and correct.

Reply:
Yep that's correct 😄